pkg/fail2ban/fail2ban.go

```diff
@@ -27,7 +27,7 @@ func New(rules rules.RulesTransformed) *Fail2Ban {
 	}
 }
 
-// ShouldAllow check if the request should be allowed.
+// ShouldAllow check if the request should be allowed. Called when a request was DENIED - increments the denied counter.
```

Comment:
To improve readability, I would put those on two lines. :)
```suggestion
// ShouldAllow check if the request should be allowed.
// Called when a request was DENIED - increments the denied counter.
```